ui/src/timeMachine/components/view_options/Affixes.tsx

```diff
@@ -2,21 +2,36 @@
 import React, {PureComponent, ChangeEvent} from 'react'
 
 // Components
-import {Form, Input, Grid} from '@influxdata/clockface'
+import {
+  Form,
+  Input,
+  Grid,
+  Toggle,
+  InputToggleType,
+  InputLabel,
+  FlexBox,
+  AlignItems,
+  ComponentSize,
+  JustifyContent
```

Comment:
hanging comma

Reply:
this is another prettier issue, as its not failing on the lint, so clearly its cool with this

Reply:
super weird, 🤷‍♂